doc/src/manual/strings.md

```diff
@@ -737,15 +737,17 @@ Some other useful functions include:
 
 ## [Non-Standard String Literals](@id non-standard-string-literals)
 
-There are situations when you want to construct a string or use string semantics, but the behavior
-of the standard string construct is not quite what is needed. For these kinds of situations, Julia
-provides [non-standard string literals](@ref). A non-standard string literal looks like a regular
-double-quoted string literal, but is immediately prefixed by an identifier, and doesn't behave
-quite like a normal string literal.  Regular expressions, byte array literals and version number
-literals, as described below, are some examples of non-standard string literals. Other examples
-are given in the [Metaprogramming](@ref) section.
-
-## Regular Expressions
+A non-standard string literal looks like a regular double-quoted string literal,
+but is immediately prefixed by an identifier, and may behave differently from a normal string literal.
+These are useful in situations when you want to construct a string or use string semantics,
+but the behavior of the standard string construct is not quite what is needed.
```

Comment:
I think I prefer the overall order of the old paragraph here (while still fixing the links)